modules/js/src/core_bindings.cpp

```diff
@@ -443,9 +448,10 @@ EMSCRIPTEN_BINDINGS(binding_utils)
     emscripten::class_<cv::Mat>("Mat")
         .constructor<>()
         .constructor<const Mat&>()
-        .constructor<Size, int>()
+        // .constructor<Size, int>()
         .constructor<int, int, int>()
         .constructor<int, int, int, const Scalar&>()
+        .constructor(&binding_utils::createMatWithDims, allow_raw_pointers())
```

Comment:
If there is a conflict with other constructors, please try class function instead.